wpilibc/shared/src/Filters/LinearDigitalFilter.cpp

```diff
@@ -7,8 +7,8 @@
 
 #include "Filters/LinearDigitalFilter.h"
 
-#include <assert.h>
-#include <math.h>
+#include <cassert>
```

Comment:
assert() is a macro, so it doesn't go in the std namespace. This file was already using the std namespace variants of the math.h functions, so there are no changes necessary.